tests/test_url_config.py

```diff
@@ -108,10 +113,91 @@ def test_bodedga_get(self):
         # Either g1 is 0, bodega changed or someone broke URLConfigs
         self.assertTrue(p.test_config)
 
-        st2 = self.st.new_context()
-        st2.set_config({'test_config': 'bodega://g1?bodega_version=v2'})
-        p2 = st2.get_single_plugin(nt_test_run_id, 'test_data')
-        self.assertEqual(p.test_config, p2.test_config)
+    @unittest.skipIf(not straxen.utilix_is_configured(), "No db access, cannot test CMT.")
+    def test_seg_file_json(self, ab_value=20, cd_value=21, dump_as='json'):
+        """
+        Test that we are getting ~the same value from interpolating at the central date in a dict
+
+        :param ab_value, cd_value: some values to test against
+        :param dump_as: Write as csv or as json file
+        """
+        central_datetime = utilix.rundb.xent_collection().find_one(
+            {'number': int(nt_test_run_id)},
+            projection={'start': 1}
+        ).get('start', 'QUERY FAILED!')
+        fake_file = {'time': [datetime(2000, 1, 1).timestamp() * 1e9,
+                              central_datetime.timestamp() * 1e9,
+                              datetime(2040, 1, 1).timestamp() * 1e9],
+                     'ab': [10, ab_value, 30],
+                     'cd': [11, cd_value, 31]
+                         }
+        if dump_as == 'json':
+            fake_file_name = 'test_seg.json'
+
+            with open(fake_file_name, 'w') as f:
+                json.dump(fake_file, f)
```

Comment:
I think you should store this file in the strax data directory of the strax storage. Joran spend quite some time to unify the data handling of all those tests, that things get up cleaned properly. So I think these tempfiles should be put there too. Or use a tempfir like in your test above which get properly cleaned even though your test may break.

Reply:
Done, used tempfile